repos/system_upgrade/el7toel8/actors/removeupgradebootentry/libraries/library.py

```diff
@@ -0,0 +1,26 @@
+from leapp.exceptions import StopActorExecutionError
+from leapp.libraries.stdlib import api, call
+from leapp.models import BootContent
+
+
+def remove_boot_entry():
+    kernel_filepath = get_upgrade_kernel_filepath()
+    call([
```

Comment:
I guess we should split this off into its own actor. Because this is failing quite often when there's something unmountable in FSTAB e.g. an NFS share. What you think? (Anyway this is not to block this PR but about kicking of the thinking) 
Mainly this is irrelevant to the task the actor does anyway, so it's misleading IMHO.

Reply:
I can see you wrote the original actor. Is the `mount -a` necessary before calling _grubby_ anyway?

Reply:
@vinzenz ping :)

Reply:
@bocekm mount -a is what should IMHO happen first thing, but maybe reversed order is better in case mount fails, there wouldn't be a reboot loop :-)